corehq/sql_db/jsonops.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
could do with a module docstring illustrating usage, particularly how they can be nested

Reply:
also, how about some tests?

Reply:
59b3dcd2fb561014a8b7c8625353d0ffb3922ee1